modules/dnn/src/dnn.cpp

```diff
@@ -191,6 +191,31 @@ void blobFromImages(InputArrayOfArrays images_, OutputArray blob_, double scalef
     }
 }
 
+void imagesFromBlob(const cv::Mat& blob_, OutputArrayOfArrays images_)
+{
+    CV_TRACE_FUNCTION();
+
+    //A blob is a 4 dimensional matrix in floating point precision
+    //blob_[0] = batchSize = nbOfImages
+    //blob_[1] = nbOfChannels
+    //blob_[2] = height
+    //blob_[3] = width
+    CV_Assert(blob_.depth() == CV_32F);
+    CV_Assert(blob_.dims == 4);
+
+    images_.create(cv::Size(1, blob_.size[0]), blob_.depth());
```

Comment:
Please add a test for new function at https://github.com/opencv/opencv/blob/master/modules/dnn/test/test_misc.cpp . I suggest you to generate several 3-channel images with `CV_32F` depth. Then create a blob using `blobFromImages` with no resize, mean subtraction, scale or channels swapping. Pass it to `imagesFromBlob` and check that output matches with origin images.